www/voll03/cv05/app/UsersDB.php

```diff
@@ -0,0 +1,238 @@
+<?php
+
+class UsersDB extends Database
+{
+    protected $dbFileName = "users";
+
+    public function __construct()
+    {
+        parent::__construct();
+        $this->dbPath = $this->dbPath . $this->dbFileName . $this->dbExtension;
+
+        if (!$this->load()) {
+            throw new Exception("*SERVER ERROR: Can't connect to database!");

```

Comment:
nekolik veci se stale opakuje, coz by slo to dat do super class, ale jste na spravne ceste